config.ini.example

```diff
@@ -1,2 +1,6 @@
 botuser = 'ExampleBot'
-botpass = 'XXXXX'
+botpass = '****'
+dbuser='ExampleDBUser'
+dbpass='****'
+db='enwiki_p'
+dbhost='enwiki.labsdb'
```

Comment:
Should be `enwiki.web.db.svc.wikimedia.cloud`

Reply:
The value of this parameter must be adapted by the user of popularpages.

Reply:
I fixed errors using phpcbf.

Reply:
> The value of this parameter must be adapted by the user of popularpages.

Yes, and in fact we need to make the bot programatically use the right db based on what wiki it's editing. I can add that logic in a different commit.

But regardless, `enwiki.labsdb` is deprecated and should not be used. See https://wikitech.wikimedia.org/wiki/Help:Toolforge/Database#Naming_conventions

Reply:
done